### PR TITLE
Add refresh btn

### PR DESCRIPTION
**Pull request by galdil**

### CODE REVIEW

File: .swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (almoghaimo) [inline .swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata:1]: What that?

File: Apester/ViewControllers/Unit/APEUnitViewController.swift [hunk 2]
```diff
@@ -24,7 +28,7 @@ class APEUnitViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
 
-        let configuration = UnitConfigurationsFactory.configurations(for: .production, hideApesterAds: false, gdprString: nil, baseUrl: "")[0]
+        let configuration = UnitConfigurationsFactory.configurations(for: .local, hideApesterAds: false, gdprString: nil, baseUrl: nil)[0]
```
Comment (almoghaimo) [inline Apester/ViewControllers/Unit/APEUnitViewController.swift:31]: Can you revert to prod?

File: .swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata (hunk 1 above)
Comment (galdil) [reply .swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata:1]: idk

**Review by almoghaimo — APPROVED** [review]
lgtm

**Pull request merged**